db/people.yml

```diff
@@ -546,3 +546,31 @@ sbanerjee:
   name: Soham Banerjee
   position: Graduate Student
   orcid_id: 0000-0001-9271-493X
+stao:
+  aka:
+    - Songsheng Tao
+    - Tao, Songsheng
+    - S. Tao
+    - Tao, S.
+  avatar: ''
+  bio: ''
+  education:
+    - begin_year: 2017
+      degree: MS/PhD in Applied Physics
+      institution: Columbia University
+      department: apam
+      group: bg
+      location: New York, NY
+      other:
+        - 'Adviser: Simon Billinge'
+    - begin_year: 2013
+      degree: BS in Physics
+      end_year: 2017
+      institution: Peking University
+      location: Beijing, China
+      other:
```

Comment:
this is not a required field, so if there is no minor, or any other "other" notes, you can just delete other and the next line.